src/main/java/seedu/elisa/model/RescheduleTask.java

```diff
@@ -82,7 +83,17 @@ public void run() {
         long period = event.getPeriod().getPeriod();
         LocalDateTime newStart = LocalDateTime.now().plusNanos(Duration.ofMillis(period).toNanos());
         Event newEvent = event.changeStartDateTime(newStart);
-        Item newItem = item.changeEvent(newEvent);
+        Item newItem;
+
+        if (item.hasReminder()) {
+            Reminder oldReminder = item.getReminder().get();
+            LocalDateTime newReminderStart = oldReminder.getOccurrenceDateTime()
+                    .plusNanos(Duration.ofMillis(period).toNanos());
+            Reminder newReminder = oldItem.getReminder().get().changeOccurrenceDateTime(newReminderStart);
+            newItem = item.changeEvent(newEvent).changeReminder(newReminder);
+        } else {
+            newItem = item.changeEvent(newEvent);
+        }
```

Comment:
Do you want to exclude this portion? Since you mentioned that autorescheduling reminders should in v2.0